.gitignore

```diff
@@ -34,9 +32,7 @@ iojs_g
 node
 node_g
 *.swp
-.benchmark_reports
 icu_config.gypi
-.eslintcache
```

Comment:
What is the reasoning for removing this one? I wouldn't expect everyone to have this in their own `.gitignore`.

Reply:
IIUC they are all covered by
https://github.com/nodejs/node/blob/c45a9da926f8de50ae42be4e9cda52fe485435dc/.gitignore#L2

Reply:
> 
> 
> IIUC they are all covered by
> 
> [node/.gitignore](https://github.com/nodejs/node/blob/c45a9da926f8de50ae42be4e9cda52fe485435dc/.gitignore#L2)
> 
> Line 2 in [c45a9da](/nodejs/node/commit/c45a9da926f8de50ae42be4e9cda52fe485435dc)
>  .*


It would be nice if that was included in the commit message for anyone glancing through the commit history.

Reply:
> It would be nice if that was included in the commit message for anyone glancing through the commit history.

@richardlau I've updated the commit message to say:

    Remove entries from `.gitignore` that are already covered by the generic
    `.*` entry on line 2 of the .gitignore file.